content/docs/reference/api/oracle/_index.md

```diff
@@ -169,7 +169,11 @@ aut contract call --address 0x47e9Fbef8C83A1714F1951F142132E6e90F5fa5D getSymbol
 
 {{< tabpane langEqualsHeader=true >}}
 {{< tab header="aut" >}}
+<<<<<<< HEAD
 aut contract call --address 0x47e9Fbef8C83A1714F1951F142132E6e90F5fa5D getSymbols
+=======
+aut contract call --address 0x47e9Fbef8C83A1714F1951F142132E6e90F5fa5D getSymbols               
+>>>>>>> b68833b (Oracle, edits to add Docker run and install)
```

Comment:
This needs to be fixed

Reply:
Fixed in commit 2246212b240621e71ca0d995cf34a15a4e61b93c.